bindings/python/test/test_viewer.py

```
@@ -260,3 +289,16 @@ def test_add_label_success(
+
+def test_compute_celestial_angular_diameter_from_states_success(
+    orbit: Orbit,
+    interval: Interval,
+    environment: Environment,
+) -> None:
+    _compute_celestial_angular_diameter_from_states(
+        celestial=environment.access_celestial_object_with_name("Sun"),
+        states=orbit.get_states_at(
+            interval.generated_grid(Duration.seconds(30.0)),
+        ),
+    ).mean() == pytest.approx(0.52, rel=1e-2)
```

Ruff (0.8.2)

Pointless comparison at end of function scope. Did you mean to return the expression result?

(B015)